squire_lib/src/ffi_rounds.rs

```diff
@@ -151,12 +151,12 @@ impl RoundId {
         match FFI_TOURNAMENT_REGISTRY.get().unwrap().get_mut(&tid) {
             Some(mut tournament) => {
                 match tournament.apply_op(TournOp::AdminRecordResult(
-                    TournOfficialId::Admin(aid),
+                    TournOfficialId::from(aid),
```

Comment:
Because `TournOfficialId` implements `From<AdminId>`, you automatically get `Into<TournOfficialId>` for `AdminId`. This lets you do the following instead:
```suggestion
                    aid.into(),
```

You can do something similar with the `PlayerIdentifier` and `RoundIdentifier`.

Reply:
Thanks for spotting this. I am getting an official lookup error for the null id though; is this expected?

```
3: [FFI]: ffi_record_result error OfficalLookup
```

Reply:
resolved in [521975d](https://github.com/MonarchDevelopment/SquireCore/pull/32/commits/521975dd0a8336c7b7d0f368507653acd627e212)

Reply:
In the future, we will have a user's account (or spoofed account if it's their first time), and we can use that account to automatically link the account as an admin. But this is a good enough work around for now.